lib/pages/caregiver/plans_page.dart

```diff
@@ -10,71 +13,104 @@ import 'package:fokus/widgets/attribute_chip.dart';
-											color: Colors.teal,
-											icon: Icons.keyboard_arrow_right,
-											onTapped: () => {log("go to details page")}
-										),
-										chips: <Widget>[
-											AttributeChip.withIcon(color: Colors.green, icon: Icons.radio_button_checked, tooltip: '$_pageKey.content.activePlan'),
-											AttributeChip.withIcon(content: "4 zadania", color: Colors.indigo, icon: Icons.layers)
-										],
-									),
-									ItemCard(
-										title: "Projekt szkolny", 
-										subtitle: AppLocales.of(context).translate('repeatability.once', {'DAY': '12 sierpnia'}),
-										actionButton: ItemCardActionButton(
-											color: Colors.teal,
-											icon: Icons.keyboard_arrow_right,
-											onTapped: () => {log("go to details page")}
-										),
-										chips: <Widget>[
-											AttributeChip.withIcon(color: Colors.red, icon: Icons.radio_button_unchecked, tooltip: '$_pageKey.content.inactivePlan'),
-											AttributeChip.withIcon(content: "2 zadania", color: Colors.indigo, icon: Icons.layers)
-										],
-									)
-								]
-							)
-						]
-					)
-				]
-			),
-			bottomNavigationBar: AppNavigationBar.caregiverPage(currentIndex: 1)
-    );
-	}
+        body: Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: <Widget>[
+              AppHeader.normal(
+                  title: '$_pageKey.header.title',
+                  text: '$_pageKey.header.pageHint',
+                  headerActionButtons: [
+                    HeaderActionButton.normal(Icons.add,
+                        '$_pageKey.header.addPlan', () => {log("Dodaj plan")}),
+                    HeaderActionButton.normal(
+                        Icons.calendar_today,
+                        '$_pageKey.header.calendar',
+                        () => {log("Kalendarz")},
+                        Colors.amber)
+                  ]),
+              CubitBuilder<CaregiverPlansCubit, CaregiverPlansState>(
+                  cubit: CubitProvider.of<CaregiverPlansCubit>(context),
+                  builder: (context, state) {
+                    if (state is CaregiverPlansInitial)
+                      CubitProvider.of<CaregiverPlansCubit>(context)
+                          .loadCaregiverPlans();
+                    else if (state is CaregiverPlansLoadSuccess)
+                      return AppSegments(
+                          segments: _buildPanelSegments(state, context));
+                    return Expanded(
+                        child: Center(child: CircularProgressIndicator()));
+                  }
+              )
+            ]
+				),
+        bottomNavigationBar: AppNavigationBar.caregiverPage(currentIndex: 1));
+  }
+}
+
+List<Segment> _buildPanelSegments(CaregiverPlansLoadSuccess state, context) {
+  var activePlans =
+      state.plans.where((blueprint) => (blueprint.isActive)).toList();
+  var deactivatedPlans =
+      state.plans.where((blueprint) => (!blueprint.isActive)).toList();
```

Comment:
Maybe try increasing the max line length so that it does not wrap everything so aggressively ;)
![image](https://user-images.githubusercontent.com/8643919/89101486-7c605d00-d400-11ea-9aaa-36342757c0fb.png)